sql/connect/server/src/main/scala/org/apache/spark/sql/connect/ml/MLCache.scala

```diff
@@ -86,7 +86,7 @@ private[connect] class MLCache extends Logging {
 
 private[connect] object MLCache {
   // The maximum number of distinct items in the cache.
-  private val MAX_CACHED_ITEMS = 100
```

Comment:
is this related?
if no, I think we should file a separate PR to increase the value